### PR TITLE
gpupdate: add page

### PR DESCRIPTION
**Pull request by skitau**
Another regular tool in the belt.

Happy Hacktoberfest!

### CODE REVIEW

File: pages/windows/gpupdate.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> A tool to check and apply Windows Group Policy settings.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/gpupdate>.
+
+- Check and apply updated Group Policy settings:
+
+`gpupdate`
+
+- Specify the target Group Policy settings to check for update:
+
+`gpupdate /Target=:{{Computer|User}}`
```
Comment (navarroaxel) [inline pages/windows/gpupdate.md:12]: `target` is in lower case in the docs, also we use `snake_case` for variables like `user`:
```suggestion
`gpupdate /target=:{{computer|user}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @skitau, just 1 comments.

**Comment by skitau** [comment]
Forgot post a reply... Thanks for the pickup @navarroaxel . Have fixed.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @skitau!

**Pull request merged**